<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(170, 170, 255);
	color: rgb(255, 255, 255);
	font-weight: bold
}
QPushButton#exit_btn{
	
	background-color: rgb(209, 0, 0);
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
	font-weight: bold

}
QDialog{

	background-color: rgb(44, 44, 64);
}
QLabel{
	
	color: rgb(255, 255, 255);
}
QLineEdit{
	
	background-color: rgb(50, 50, 64);
	color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QLabel" name="main_lbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
  <widget class="QLabel" name="lgn_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ваш логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lgn_line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pwd_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ваш пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reg_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lgn_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти из приложения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
